src/components/bs5/header/header.hbs

```diff
@@ -273,12 +273,11 @@
                                                     {{/isType}}
 
                                                     {{#isType actionType "select"}}
-                                                        <select class="qld-text-input form-select qld-select" aria-label="" required=""
-                                                            aria-required="">
-                                                                <option selected="">Please select</option>
-                                                                {{#each options}}
-                                                                    <option value="{{value}}">{{value}}</option>
-                                                                {{/each}}
+                                                        <select class="qld-text-input form-select qld-select {{actionAtts.classes}}" id="{{actionAtts.id}}" name="{{actionAtts.name}}" {{actionAtts.required}} aria-required="{{actionAtts.required}}" aria-label="{{actionAtts.ariaLabel}}" >
+                                                            <option selected="">Please select</option>
+                                                            {{#each options}}
+                                                                <option {{selected}} {{disabled}} value="{{key}}">{{value}}</option>
+                                                            {{/each}}
                                                         </select>
                                                     {{/isType}}
 
```

Comment:
Hey @MitchAnderson93, we might need to pass some custom HTML in this menu flyout for ForGov. I started to add the additional markup we needed, but the structure around the agency select is a bit too complex to use JSON variables. 

What are your thoughts on an additional actionType called "custom" in the groups object, and a new block in the template?

`{{#isType actionType "custom"}}
  {{ customContent }}
  {{/isType}}
`